sdk/communication/communication-jobrouter/README.md

```diff
@@ -0,0 +1,360 @@
+| Distribution Policy | The distribution policy is a container of rules on how jobs are allocated and distributed to workers from a queue. This includes offer time to live, distribution method (e.g. round robin, top (n) available agents, etc) and allocation rules. |
+| Exception Policy | A container of rules that define what action to take when a particular exception occurs.  This container of rules can be associated with one or more queues. |
+
+##Setting Up
+###Install an IDE 
+Install IDE such as [VSCode](https://code.visualstudio.com/download) or [Webstorm](https://www.jetbrains.com/webstorm/download/) if you haven't (Optional)
+
+###Install NodeJS if you haven't
+LTS or Current version, either is ok.
+https://nodejs.org/en/about/releases/
+
+### Create a new NodeJS Express server
+
+In a console window (such as cmd, PowerShell, or Bash), create a new folder named `RouterQuickStart` and use `npx express-generator` to create a new Express app in the current folder. This will create a simple "Hello World" Express project that will listen on port `3000`, as follows:
+
+```
+mkdir RouterQuickStart
+cd RouterQuickStart
+npx express-generator
+npm install
+DEBUG=routerquickstart:* npm start
+```
+
+###Create an ACS resource under dogfood
+Refer to this doc, https://skype.visualstudio.com/SPOOL/_wiki/wikis/SPOOL.wiki/20243/Azure-dogfood-env
+Create a new resource or use existing resource. If creating a resource, select `acs-pstn-dogfood` as resource group.
+
+![image.png](/.attachments/image-1a0118bb-0063-420b-924a-943b66ef0d01.png)
+
+###Install the Azure Communication Services Router SDK
+In the application directory, install the Azure Communication Services Router client library for JavaScript package by using the `npm install --save` command
+`npm install --save @azure/communication-jobrouter`
+
+## Examples
+###Initialize Router Client
+First we need to initialize a Router client.
+```js
+import { RouterClient } from '@azure/communication-jobrouter';
+
+const acsConnectionString = 'endpoint=https://<YOUR_ACS>.communication.azure.com/;accesskey=<YOUR_ACCESS_KEY>'
+const routerClient = new RouterClient(acsConnectionString );
+const routerAdministrationClient = new RouterAdministrationClient(acsConnectionString );
+```
+
+##Configure Queue and Workers
+###Create a queue
+Then we create a sales queue with various labels.
+```js
+const salesQueueResponse = await routerAdministrationClient.CreateQueue({
+  "name": "Sales",
+  "distributionPolicyId": "0832f57a-a651-4cd3-b721-1a1ce76ecf2b",
+  "labels": {
+    "Department": "Xbox"
+  },
+  "exceptionPolicyId": "cb272ee4-38ae-41d8-9408-190d9c7a98aa"
+});
+```
+
+###Register workers
+Register workers “Bob” and “Alice” with various abilities.
+```js
+// Create worker Bob
+const workerId = "21837c88-6967-4078-86b9-1207821a8392";
+const bobWorkerResponse = await routerClient.CreateWorker(workerId, {
```

Comment:
These operation names don't seem to match with what the package exports, this is PascalCase